Rubberduck.Main/Extension.cs

```diff
@@ -145,7 +145,7 @@ private void InitializeAddIn()
                     return;
                 }
 
-                var configLoader = new XmlPersistanceService<GeneralSettings>
+                var configLoader = new XmlPersistanceService<GeneralSettings>()
```

Comment:
Redundant parens?

Reply:
That was from my aborted attempt to ctor-inject which caused more problems than solutions. Will remove.